sardana_tango/ctrl/TangoAttrZeroDCtrl.py

```diff
@@ -91,28 +91,28 @@ def read_all(self):
                     self._log.error('Exception reading attributes:%s.%s',
                                     dev, str(attributes))
 
-            for attr in attributes:
-                axies = []
-                for axis, dic in self.devsExtraAttributes.items():
-                    if dic[TANGO_ATTR] == dev+'/'+attr:
-                        axies.append(axis)
-                for axis in axies:
-                    if len(values) > 0:
-                        dev_attr_value = values[attr]
-                        if dev_attr_value.has_failed:
+                for attr in attributes:
+                    axies = []
+                    for axis, dic in self.devsExtraAttributes.items():
+                        if dic[TANGO_ATTR] == dev+'/'+attr:
+                            axies.append(axis)
+                    for axis in axies:
+                        if len(values) > 0:
+                            dev_attr_value = values[attr]
+                            if dev_attr_value.has_failed:
                             # In case of Attribute error
```

Comment:
@teresanunez, just a cosmetic change, but flake8 complains that this line (comment) should be indented:
> ./sardana_tango/ctrl/TangoAttrZeroDCtrl.py:103:29: E115 expected an indented block (comment)

I would have fixed it myself but I don't have permission to push to your fork.

Reply:
I merge it and then will make the correction directly on the master branch.